hoomd/box.py

```diff
@@ -32,24 +30,24 @@ class (e.g. Scalar3, Int3).
         except (ValueError, TypeError):
             raise ValueError("Expected values of type {}.".format(scalar_type))
     else:
-        raise ValueError("Expected three or one value received {}.".format(
-            len(vec)))
+        raise ValueError("Expected a sequence of three values or a single "
+                         "value. Received {} values.".format(len(vec)))
 
 
-_make_scalar3 = partial(_make_vec_three,
+_make_scalar3 = partial(_make_vec3,
                         vec_factory=_hoomd.make_scalar3,
                         scalar_type=float)
 
-_make_int3 = partial(_make_vec_three,
+_make_int3 = partial(_make_vec3,
                      vec_factory=_hoomd.make_int3,
                      scalar_type=int)
 
-_make_char3 = partial(_make_vec_three,
+_make_char3 = partial(_make_vec3,
                       vec_factory=_hoomd.make_char3,
                       scalar_type=int)
 
 
-def _to_three_array(vec, dtype=None):
+def _vec3_to_array(vec, dtype=None):
```

Comment:
Including `vec3` in the name of this conversion function makes it slightly clearer what the function does, and it makes it clear that this is (nearly) the inverse of `_make_vec3`.